10_date_timestamps_and_lies.md

```diff
@@ -111,7 +111,7 @@ I feel silly saying this, by I've tried casting to string which doesn't seem to
 Actually I tought that the solution works but...
 
 ```scala
-df.filter(columns.isNull)
+df.filter($"lastUpdate".isNull)
```

Comment:
We could probably drop it to whatsoever. It is already illustrated by the subsequent code.